### PR TITLE
Only call `org.bukkit.World` height methods if they're present (1.17+)

### PR DESCRIPTION
**Pull request by dordsor21**

### CODE REVIEW

File: worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitWorld.java [hunk 1]
```diff
@@ -82,10 +82,10 @@
 
 public class BukkitWorld extends AbstractWorld {
 
-    private static final Logger LOGGER = LogManagerCompat.getLogger();
+    public static final boolean HAS_3D_BIOMES;
+    public static final boolean HAS_MIN_Y;
```
Comment (NotMyFault) [inline worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitWorld.java:86]: Missing Fawe annotations, but `HAS_3D_BIOMES` can stay private, no?

**Review by NotMyFault — APPROVED** [review]
Untested, due me not having such an old test setup atm, but looks good.

**Review by SirYwell — CHANGES_REQUESTED** [review]
Wouldn't it make more sense to use the `getMinY()`|`getMaxY()` methods from the WE world? (or `getMinimumPoint().getY()`|`getMaximumPoint().getY()` from Extent if no world is present)

**Comment by dordsor21** [comment]
> Wouldn't it make more sense to use the `getMinY()`|`getMaxY()` methods from the WE world? (or `getMinimumPoint().getY()`|`getMaximumPoint().getY()` from Extent if no world is present)

Changing to a BukkitWorld for regen stuff would require wrapping or the method signatures to all be changes to accept BukkitWorlds. For that case it's far much efficient to just access the static field

**Review by SirYwell — APPROVED** [review]

**Comment by NotMyFault** [comment]
This PR is ready to be merged, we will merge it in ~24 hours if there is no negative feedback, as documented by the [merge process](https://github.com/IntellectualSites/IntellectualSites/blob/main/docs/MAINTAINERS_CONTRIBUTORS.adoc#merge-process) documentation.

**Pull request merged**